libraries/AP_OSD/AP_OSD.h

```diff
@@ -278,6 +456,14 @@ class AP_OSD
         _disable = false;
     }
 
+    AP_OSD_AbstractScreen& get_screen(uint8_t idx) {
+        return (idx >= AP_OSD_NUM_DISPLAY_SCREENS) ? (AP_OSD_AbstractScreen&)param_screen[idx - AP_OSD_NUM_DISPLAY_SCREENS] : (AP_OSD_AbstractScreen&)screen[idx];
+    }
+
+    // Check whether arming is allowed
+    bool pre_arm_check(char *failure_msg, const uint8_t failure_msg_len) const;
```

Comment:
We should really canonicalise on one of `pre_arm_check` or `pre_arm_checks`

Reply:
Which one do you prefer? They seem pretty evenly distributed through the code. Maybe slightly more pre_arm_check()s